types/nightwatch/index.d.ts

```diff
@@ -1493,6 +1492,77 @@ export function element(locator: string | ElementProperties | By | WebElement, o
 
 export type NightwatchTests = NightwatchTestFunctions | NightwatchTestHooks;
 
+export class DescribeInstance {
+    '[instance]': any;
+    '[attributes]': {};
+    '[client]': NightwatchClient;
+    name(): any;
+    tags(): any;
+    unitTest(): any;
+    endSessionOnFail(): any;
+    skipTestcasesOnFail(): any;
+    disabled(): any;
+    desiredCapabilities(): any;
+    page(): any;
+    globals(): any;
+    settings(): any;
+    argv(): any;
+    timeout(value: any): void;
```

Comment:
Can we use more specific types for the parameters other than any? For example, `number` here.

Reply:
Fixed in latest commits